### ISSUE TITLE
HABmin does not load if http port is 80

### ISSUE THREAD

**Issue by CreaturePH** [openhab/org.openhab.ui.habmin#269]
When I change the port openhab from 8080 to 80 habmin stops working. Constantly displays "Loading ..." and nothing else.

**Comment by cdjackson** [comment]
Does port 80 really work? Often this is blocked in Linux and is only available for system use. Does the rest interface work? Does PaperUI work?

**Comment by CreaturePH** [comment]
Yes, all other services on this port are working correctly. As soon as I change the port back to 8080 habmin starts working again.

**Comment by cdjackson** [comment]
Ok, I'm not sure why this would be as HABmin doesn't set the port. Please provide a log or error report or something from which we can see what's happening.

**Comment by CreaturePH** [comment]
I started debug_start.sh after that again tried to open the habmin. There is nothing in the logs (userdata/logs/openhab.log), there is nothing in the console either. Where else can I look?

**Comment by cdjackson** [comment]
What console so you mean?  The JavaScript console or the oh console?
